src/EnergyPlus/SimulationManager.cc

```diff
@@ -1951,6 +1951,7 @@ namespace SimulationManager {
 					ShowWarningError( "Node Connection errors shown during \"fatal error\" processing may be false because not all inputs may have been retrieved." );
 					WarningOut = false;
 				}
+				if (CompSets(Count).CType == "COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE") continue;
```

Comment:
We have to skip this error handling because ASIHP has a different node connection style, which cause fatal error here.